### ISSUE TITLE
Not Invalidating token properly

### ISSUE THREAD

**Issue by newkillerbeast2017** [krisanalfa/lumen-jwt#20]
I checked out the package, made no changes, connected to one of my existing databases and tested the following routes:

1. POST - `api/auth/login` - token got generated
2. DELETE - `api/auth/invalidate` - token invalidated
3. GET - `api/auth/user` - expired token yet gets the data properly

Isn't it supposed to say unauthorized?

**Comment by krisanalfa** [comment]
Hi, thanks for reporting this issue. What HTTP code did you get? Sorry, but I can't reproduce this bug.

**Comment by newkillerbeast2017** [comment]
Its not an error code. What I meant is, I login a user - a token is generated. I invalidate the user - a token should be blacklisted. I get the user details - still shows data whereas the token was expired earlier and had to be shown an error message. If you still cant get me, try the login, then use the generated token to invalidate. Next, use same token to try get user. It should not give user details but whereas it does.

**Comment by newkillerbeast2017** [comment]
@krisanalfa were you able to reproduce this? Understood my issue?

**Comment by newkillerbeast2017** [comment]
@krisanalfa were you able to reproduce?

**Comment by krisanalfa** [comment]
Hi @newkillerbeast2017, can u use [this](https://www.getpostman.com/collections/1090b93eaa1ece4b09f2) postman collection to reproduce the issue? I can't reproduce.

**Comment by rbnzdave** [comment]
Yeah I also don't have the ability to expire the token. the test for it is only checking for a response of 200, so fine, but check invalid token doesn't have an issue. Running authentication and then running invalidate doesn't affect the ability to view any protected material. It seems then then expire after a "while" when it complains of missing segments until authenticated again.

**Comment by TakesTheBiscuit** [comment]
I can't reproduce this

**Comment by krisanalfa** [comment]
Hi @rbnzdave it's been a while. Sorry for late reply. How's your issue status? Have you resolved this? Until now I can't reproduce this issue, so I cannot make any fix or help you.
